libs/platform/ebpf_platform.h

```diff
@@ -517,6 +517,19 @@ extern "C"
 
     typedef struct _ebpf_hash_table ebpf_hash_table_t;
 
+    typedef enum _ebpf_hash_table_notification_type
+    {
+        EBPF_HASH_TABLE_NOTIFICATION_TYPE_ALLOCATE, //< A key + value have been allocated
+        EBPF_HASH_TABLE_NOTIFICATION_TYPE_FREE,     //< A key + value have been freed
+        EBPF_HASH_TABLE_NOTIFICATION_TYPE_USE,      //< A key + value have been used
```

Comment:
```suggestion
        EBPF_HASH_TABLE_NOTIFICATION_TYPE_ALLOCATE, //< A key + value have been allocated.
        EBPF_HASH_TABLE_NOTIFICATION_TYPE_FREE,     //< A key + value have been freed.
        EBPF_HASH_TABLE_NOTIFICATION_TYPE_USE,      //< A key + value have been used.
```
for consistency with rest of file.